anago

```diff
@@ -206,7 +206,7 @@ check_prerequisites () {
     fi
   done
   # Ensure $USER is active to start
-  if ! [[ "$gcloud_auth_list" =~ -\ $useratgoog\ \(active\) ]]; then
```

Comment:
the less fragile way to do this is shown in https://github.com/kubernetes/kubernetes/pull/30008/files